package.json

```diff
@@ -65,7 +65,11 @@
     "rollup-plugin-sourcemaps": "^0.5.0",
     "tslint": "^5.11.0",
     "tslint-config-prettier": "^1.13.0",
-    "tslint-plugin-prettier": "^1.3.0"
+    "tslint-plugin-prettier": "^1.3.0",
+    "typescript": "^4.5.5"
+  },
+  "peerDependencies": {
+    "typescript": "^4.0.3"
```

Comment:
@wagenet I may be wrong, but this not gonna make any difference as still will install `typescript@4.0.3` by npm 7+

It's technically the same as replacing `"typescript": "~4.0.3"` with `"typescript": "^4.0.3"`.

To truly decouple `typescript` version from `broccoli-typescript-compiler`, maybe you meant to make this

```json
"peerDependencies": {
    "typescript": "*"
},
"peerDependenciesMeta": {
  "typescript": {
    "optional": true
  }
},
```

Reply:
@SergeAstapov ah, you may be correct.

Reply:
My understanding is that this means that, if a newer compatible version is installed, it will use that one. So this _should_ be correct. I've been having some trouble validating it in practice either way.